### PR TITLE
Feat/mn 306/common UI web

### PR DESCRIPTION
**Pull request by rohitsharma120582**
## Technical description
This is a collection of reusable and customizable UI components for React web applications. It is designed to help developers easily build modern and responsive user interfaces.

### CODE REVIEW

**Review by deepakglobant — APPROVED** [review]

File: packages/common-ui-web/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function App() {
+  return (
+    <div>
+      <SVGElement ... />
+      <SightOverlay ... />
+    </div>
+  );
+}
+```
+
```
Comment (souyahia-monk) [inline packages/common-ui-web/README.md:26]: Add a section header here :

# API

Comment (souyahia-monk) [inline packages/common-ui-web/README.md:26]: Before `## SVGElement` please add a subsection header like this :

`# Available components`

Comment (rohitsharma120582) [reply packages/common-ui-web/README.md:26]: Done

File: packages/common-ui-web/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { SVGElement } from '@monkvision/common-ui-web';
+
+function App() {
+  return (
+    <div>
+      <SVGElement svg="" />
+    </div>
+  );
+}
+```
```
[the remark this answers is not in the document]
Comment (rohitsharma120582) [reply packages/common-ui-web/README.md]: Done

**Pull request merged**